projects/packages/changelogger/src/Utils.php

```diff
@@ -149,33 +149,51 @@ public static function loadChangeFile( $filename, &$diagnostics = null ) {
 	}
 
```

Comment:
So, technically, since this is only a minor version bump we should probably keep a stub `getTimestamp()`.
```suggestion

	/**
	 * Get a timestamp for a file.
	 *
	 * @deprecated since $$next-version$$
	 * @param string               $file File.
	 * @param OutputInterface      $output OutputInterface to write debug output to.
	 * @param DebugFormatterHelper $formatter Formatter to use to format debug output.
	 * @return string|null
	 */
	public static function getTimestamp( $file, OutputInterface $output, DebugFormatterHelper $formatter ) {
		return self::getRepoData( $file, $output, $formatter )['timestamp'];
	}

```
On the other hand, it's not like I expect anyone has any code anywhere depending on this internal function within Changelogger. 🤷

Reply:
Totally slipped my mind, thanks! Might as well leave the deprecated function around for now.